JSDemos/Demos/List/GroupedList/description.md

```diff
@@ -1 +1,11 @@
-The **List** UI component can display items in&nbsp;groups if&nbsp;they are arranged so&nbsp;in&nbsp;the data source. Groups have captions, which are customized in&nbsp;this demo.
\ No newline at end of file
+It is possible to display the **List** in groups under the following conditions:
+
+- The **List** data passed to the **dataSource** is structured so that each group is an object that contains two fields: **key** and **items**. The **key** field represents a group header, and the **items** field represents an array of group items. Refer to this [help guide](/Documentation/Guide/UI_Components/List/Grouping/In_the_Data_Source/) for more information on grouping in Data Source.   
+
+In cases when the data source doesn't conform to the structure specified above, use the [DataSource](https://js.devexpress.com/Documentation/ApiReference/Data_Layer/DataSource/)'s [group](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#group) property to define the data field to group against. If data objects are grouped but use other field names, implement the DataSource's [map](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#map) function to create **key** + **items** field mappings. Refer to this [help guide](/Documentation/Guide/Data_Binding/Data_Layer/#Reading_Data/Data_Transformation/Item_Mapping) for more information on item mapping. 
```

Comment:
I suggest removing the link to the Grouping in the Data Source article because it doesn't add anything to the things we say in this description. Also, I changed the last sentence a bit - it is better to give the name of the article you give a link to instead of just "this article".

```suggestion
- The **List** data passed to the **dataSource** is structured so that each group is an object that contains two fields: **key** and **items**. The **key** field represents a group header, and the **items** field represents an array of group items.

    In cases when the data source doesn't conform to the structure specified above, use the [DataSource](https://js.devexpress.com/Documentation/ApiReference/Data_Layer/DataSource/)'s [group](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#group) property to define the data field to group against. If data objects are grouped but use other field names, implement the DataSource's [map](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#map) function to create **key** + **items** field mappings. Refer to the following help topic for more information: [Item Mapping](/Documentation/Guide/Data_Binding/Data_Layer/#Reading_Data/Data_Transformation/Item_Mapping).
```